spa/src/components/common/Button/PublishButton/PublishButton.tsx

```diff
@@ -69,6 +70,8 @@ function PublishButton({ className }: PublishButtonProps) {
     handleOpen: handleOpenUnpublishModal
   } = useModal();
   const [anchorEl, setAnchorEl] = useState<HTMLElement | null>(null);
```

Comment:
Same here:

```suggestion
  const [anchorEl, setAnchorEl] = useState<HTMLElement>();
```

It will be undefined instead of null, but I don't think this is an issue?

Reply:
Don't disagree, but this is otherwise untouched code, so would prefer not to make change.